generated/nirfmxspecan/nirfmxspecan.proto

```diff
@@ -5614,8 +5620,8 @@ message IMFetchIntermodMeasurementRequest {
 message IMFetchIntermodMeasurementResponse {
   int32 status = 1;
   int32 intermod_order = 2;
-  double lower_intermod_power = 3;
-  double upper_intermod_power = 4;
+  double lower_intermod_absolute_power = 3;
```

Comment:
This is a source breaking change. Existing clients will still be compatible. But when they pull in the new proto file and re-protoc, they will most likely need to make source changes. If they are using a typed language (or using a type checker) this will show up as a compile error. In a dynamically typed language it will be a runtime error.

Is that OK to the RFmx team? Is it OK to your clients?

I will usually allow these because it's more of an inconvenience than a functional problem and we haven't gotten pushback on it. But it's definitely not preferred.

FYI @maxxboehme @reckenro

Reply:
Right, we've mostly left it to the driver teams to make that call and like Alex said there hasn't been pushback so far. If we do decide to continue with changing the field names we'll just need to add the "source-breaking" label to this PR. That will make it easier to note in the next release.

Reply:
We have decided not to rename the parameters.